Utilities/GenerateDocs/docker/cmd.sh

```diff
@@ -20,4 +22,16 @@ cmake \
 ninja SimpleITK-doc
 
 cd SimpleITK-build/Documentation/
+
 tar -zcvf ${OUT_DIR}/simpleitk_doxygen_html_${SIMPLEITK_GIT_TAG}.tar.gz html
+rm ${here}/SimpleITK/Wrapping/R/Packaging/SimpleITK/man/*.Rd
+
+cd ${here}/SimpleITK/Utilities/GenerateDocs/
+cp /work/bin/config_vars.sh .
+./SwigDocUpdate.sh
```

Comment:
This is a tricky one... This script is coming from the checkout repo to build SimpleITK... How can we test edits to the script?